src/pages/ClusterPage/info.js

```diff
@@ -388,12 +391,14 @@ export default class Clusters extends Component {
 				mark.plan.endsWith(this.state.cluster.pricing_plan) ||
 				mark.plan.startsWith(this.state.cluster.pricing_plan),
 		);
-
+		const { cluster, deployment } = this.state;
 		return (
 			<Fragment>
 				<FullHeader
 					isCluster
 					cluster={this.props.match.params.id}
+					clusterDetails={cluster}
+					deployment={deployment}
```

Comment:
Instead of passing these values we can just parse the required value and add it over here as a prop.